packages/phrase/src/pull-translations.test.ts

```diff
@@ -33,21 +35,21 @@ describe('pull translations', () => {
     beforeEach(() => {
       (pullAllTranslations as jest.Mock).mockClear();
       (writeFile as jest.Mock).mockClear();
-    });
-    (pullAllTranslations as jest.Mock).mockImplementation(() =>
-      Promise.resolve({
-        en: {
-          'hello.mytranslations': {
-            message: 'Hi there',
+      (pullAllTranslations as jest.Mock).mockImplementation(() =>
+        Promise.resolve({
+          en: {
+            'hello.mytranslations': {
+              message: 'Hi there',
+            },
           },
-        },
-        fr: {
-          'hello.mytranslations': {
-            message: 'merci',
+          fr: {
+            'hello.mytranslations': {
+              message: 'merci',
+            },
           },
-        },
-      }),
-    );
+        }),
+      );
+    });
```

Comment:
Pulled these mock implementations into `beforeEach` because the mock implementation for the new test was polluting the implementation for the existing tests.